tests/src/testUnified.cpp

```diff
@@ -29,7 +29,10 @@ struct ProcessArrayFactoryBackend : DeviceBackendImpl {
 
   void open() override { _opened = true; }
 
-  void close() override { _opened = false; }
+  void close() override {
+    _opened = false;
+    _pv.clear();
```

Comment:
I am surprised that this works. Accessors must survive the closing and re-opening of the backend. I thought the test even does that.

Reply:
Obviously it doesn't.

Reply:
> Obviously it doesn't.

Indeed. I am not sure if the spec says something about it. But a test might come up with something like that. You can get accessors before open, and it is not stated that you have to renew them after a close.

Reply:
removed, changed to just keep all PVs